BLSQ/iaso
# Remove iaso_beneficiaries from the database

https://bluesquare.atlassian.net/browse/IA-1632
## Self proof reading checklist

- [x ] Did I use eslint and black formatters
- [ x] Is my code clear enough and well documented
- [ x] My migrations file are included
## Changes
Migration to remove iaso_beneficiaries from the db

## How to test

Explain how to test your PR.
Run the migration, check if iaso_beneficiaries disappeared from the database
## Print screen / video
The last entry here should disappear
<img width="623" alt="CleanShot 2022-11-14 at 07 43 08@2x" src="https://user-images.githubusercontent.com/185797/201592641-f3446b57-c6a5-49b1-9fd2-359eaf0532c6.png">
